### DISCUSSION TITLE
Current install instructions?

### DISCUSSION THREAD

**Discussion by ADigitalFrontier** [mnit-rtmc/iris#139] [General]
Hello,

I've been trying to set up Iris for a while now, and have tried various approaches, yet the installation instructions (depending on which I am reading) seem to be either incomplete, outdated, or simply not working. I initially tried the setup using Mapnik only to find Mapnik is outdated (at least the Fedora-compatible version). I eventually found the bulb readme buried within the repository; however, after following that through, I get an Nginx 403 error when trying to access 127.0.0.1/iris. Maybe there is something I am missing, but I was hoping somebody could point me in the right direction for a currently working setup guide.

**Comment by DougLau** [comment]
I agree setting up mapping is in a difficult state.  The old raster tile method described [here](https://mnit-rtmc.github.io/iris/mapping.html) is something I haven't tried in a long time -- so long that I cannot offer any advice.  There might be some useful information in issue #40 that can help.

The bulb directory contains the new [web UI](https://github.com/mnit-rtmc/iris/issues/101), which is progressing nicely, but still alpha-quality.  That is why the documentation is not really finished yet.
